pilot/pkg/model/push_context.go

```diff
@@ -148,6 +157,8 @@ type Gateway struct {
 type processedDestRules struct {
 	// List of dest rule hosts. We match with the most specific host first
 	hosts []host.Name
+	// Map of dest rule host to the list of namespaces to which this destination rule has been exported to
+	exportTo map[host.Name]map[visibility.Instance]bool
```

Comment:
nit: `map[host.Name]map[visibility.Instance]bool` -> `ap[host.Name]map[visibility.Instance]struct{}` ?

Reply:
This made it easier to write the code :P than using the empty struct that requires existence checks..

Reply:
Actually, the real reason was that the service object's exportTo is map with bool as value. And even within push context, there are lots of places that use map[visibility]bool like https://github.com/istio/istio/pull/24443/files#diff-ee39cd332931ab6f8c10713ce3446d09R1204 [ps.defaultVirtualServiceExportTo].

So to keep things uniform, I maintained the same format.

Reply:
Sure. No problem